lmdeploy/lite/apis/calibrate.py

```diff
@@ -179,6 +179,8 @@ def calibrate(model: str,
     work_dir.mkdir(parents=True, exist_ok=True)
     calib_ctx.export(work_dir)
 
+    return model, tokenizer, work_dir
```

Comment:
Could update the docsting of `Returns` as well.

Reply:
Done